torch/onnx/symbolic.py

```diff
@@ -412,10 +412,9 @@ def chunk(g, self, chunks, dim):
     return split(g, self, split_size, dim)
 
 
-@parse_args('v', 'i', 'i')
+@parse_args('v', 'i', 'v')
 def select(g, self, dim, index):
-    slice_node = g.op("Slice", self, axes_i=[dim], starts_i=[index], ends_i=[index + 1])
-    return g.op("Squeeze", slice_node, axes_i=[dim])
+    return g.op("Gather", self, index, axis_i=dim)
```

Comment:
Can we check dim, if dim is 0 or 1, just emit `Gather`, otherwise use the old logic.

Reply:
I think that should go in the caffe2 backend, since it's an implementation detail of caffe2 rather than something relating to the ONNX spec. But I can do it